common/src/main/java/com/wynntils/models/players/PlayerModel.java

```diff
@@ -75,13 +92,15 @@ public Stream<String> getAllPlayerNames() {
     }
 
     public void reset() {
-        errorCount = 0;
+        errors.clear();
+        userFailures.clear();
     }
 
     @SubscribeEvent
     public void onWorldStateChange(WorldStateEvent event) {
         if (event.getNewState() == WorldState.NOT_CONNECTED) {
             clearUserCache();
```

Comment:
Do we really need to do this? I understand we need to clear the ghost cache if we change world, but the user cache (and the without-account cache) are about what information Athena has about a player, and is independent on the world.

Reply:
Thought about this. Historically we cleared it so the users get fresh cosmetics without full restarts. Maybe we should just require a restart...

Reply:
Changed it so we don't clear it.